cpp/velox/tests/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+find_arrow_lib(${PARQUET_LIB_NAME})
```

Comment:
Why need this find?

Reply:
I also refactor the test CMakeList here https://github.com/oap-project/gluten/pull/1082

Reply:
adding `CMakeLists.txt` for dir tests makes compiling tests more conveniently

Reply:
We may can remove libvelox.so from function `add_test_case`